src/app/modules/photo-uploader/services/upload-photo-file-item.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { FileItem, FileUploader } from 'ng2-file-upload';
+import { PhotoUploaderModel } from 'src/app/models/photo-uploader-model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UploadPhotoFileItemService {
+
+  constructor() { }
+
+  // tslint:disable-next-line:max-line-length
+  checkIfAddingPhotoIsNotCurrentlyExistingInPhotoUploaderQueue(fileItems: FileItem[], photoUploaderModel: PhotoUploaderModel, lastCurrentPhoto: FileItem) {
+    const foundFiles = fileItems.filter(x => x.file.name === lastCurrentPhoto.file.name && x.file.size === lastCurrentPhoto.file.size);
+    const isTheSameFilesWasUploaded = foundFiles.length >= 2;
```

Comment:
I encourage You to put number **2** to const item instead of using magic values ;)